tlog-cosignature.md

```diff
@@ -95,10 +99,23 @@ log identified by the origin line has the specified root hash.
 Extension lines MAY be included in the checkpoint by the log, and if present
 MUST be included in the cosigned message. However, it's important to understand
 that the witness is asserting observation of correct append-only operation of
-the log based on the first three lines of the checkpoint; no semantic statement
-is made about any extension lines, and consensus between witnesses on the
-extension lines SHALL NOT be assumed.
+the log based on the first three lines of the checkpoint; consensus between
+witnesses on the extension lines SHALL NOT be assumed, and no semantic statement
+is made about any extension lines unless the witness operator says otherwise.
+
+A witness operator MAY make additional statements about a checkpoint.  These
+additional statements need to be communicated out of band to those defining
+trust policies based on tuples of (public key, supported cosignature version).
+The same tuple MUST NOT be reused across different cosigning contexts.
```

Comment:
"different cosigning contexts" is potentially ambiguous; from our discussions I know that a cosigning context fixes a set of claims that can be relied on. But this sentence could be interpreted to mean something like "each key should only be used for cosigning a single log".

Reply:
Pulled the trigger before reading the next sentence...